django/__main__.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+"""
+Alias to allow the django module to be run as a script.
+
+Example:
+
+    python -m django collectstatic
+"""
+from django.core import management
+
+if __name__ == "__main__":
```

Comment:
Isn't that a noop? This makes sense for normal console scripts, but here it will always be true, no?